i18n.config.json

```diff
@@ -5,550 +5,619 @@
     "name": "English",
     "localName": "English",
     "langDir": "ltr",
-    "dateFormat": "MM/DD/YYYY"
+    "dateFormat": "MM/DD/YYYY",
+    "validISO639_1": true
```

Comment:
curious why include this for all instead of just adding `invalidISO0639_1` to `pcm` and `fil`?

Reply:
I prefer a consistent scheme for better typing, and usually go with positive names for clearer logic. That said, I'm not strongly attached to it. Happy to switch to your suggestion if you feel strongly about it.